tool/SignalGenerator/signalgenerator.h

```diff
@@ -39,6 +39,13 @@ namespace daq::streaming_protocol::siggen {
     static const char EXECUTIONTIME[] = "executionTime";
     static const char PROCESSPERIOD[] = "processPeriod";
 
+    static const char RANGE[] = "range";
+    static const char LOW[] = "low";
+    static const char HIGH[] = "high";
+    static const char POSTSCALING[] = "postScaling";
+    static const char SCALE[] = "scale";
+    static const char POFFSET[] = "poffset";
```

Comment:
Shouldn't this be called "offset". This is also already defined in "Defines.h"

Reply:
I've replaced local constants with the ones prefixed "META_*" from "Defines.h"